orchestrator_builder.sh

```diff
@@ -107,7 +107,7 @@ get_sources(){
     REVISION=$(git rev-parse --short HEAD)
     echo "REVISION=${REVISION}" >> ${WORKDIR}/orchestrator.properties
     wget https://raw.githubusercontent.com/percona/orchestrator-packaging/master/29591_orc-8026.patch
-    git apply 29591_orc-8026.patch
+    patch -p1 < 29591_orc-8026.patch
     git diff
```

Comment:
it is useless in this case

Reply:
Could you suggest the correct command?

![Screenshot 2021-12-03 at 15 18 39](https://user-images.githubusercontent.com/25184101/144609186-b4c452a3-33ab-4e77-a4e2-6d4a2cc6bd5c.png)